subprojects/cli11.wrap

```diff
@@ -0,0 +1,7 @@
+[wrap-git]
+url = https://github.com/CLIUtils/CLI11.git
+revision = v1.9.1
```

Comment:
Does it have release tarball? We currently don't allow git, in personally not against lifting that requirement but it's controversial ATM.

Reply:
Yeah. I'm currently working on replacing the wrap-git file with a wrap-file in my personal project. I'll create another PR once I have it working.

Reply:
You can just force push to your pr when you edit the commit instead of creating a new one.